anki-remastered-api/src/main.ts

```diff
@@ -0,0 +1,12 @@
+import {NestFactory} from '@nestjs/core';
+import {AppModule} from './app.module';
+import {ValidationPipe} from '@nestjs/common';
+import "reflect-metadata";
+
+async function bootstrap() {
+    const app = await NestFactory.create(AppModule);
+    app.useGlobalPipes(new ValidationPipe({stopAtFirstError: true}));
+    await app.listen(process.env.PORT ?? 3000);
```

Comment:
app.enableCors({
origin: true,
methods: ['GET', 'POST', 'PUT', 'DELETE', 'PATCH'],
});

need this for the frontend